### ISSUE TITLE
For purposes of meeting the nested requirements of SC 4.1.1 can ARIA roles be used?

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#1511]
Say I have a structure such as

```
<div role="list">
  <li>Apples</li>
  <li>Bananas</li>
</div>
```

This would fail the W3C validator because the list items are not nested in a list - but because the role of list changes the div to a list it should pass.  Does this meet SC 4.1.1?

**Comment by patrickhlauke** [comment]
Personally, I'd pass this, as I don't think any user agents have trouble parsing and correctly building a tree out of this

> The intent of this Success Criterion is to ensure that user agents, including assistive technologies, can accurately interpret and parse content. If the content cannot be parsed into a data structure, then different user agents may present it differently or be completely unable to parse it.

**Comment by fstrr** [comment]
This issue looks like it can be closed. If it needs to be re-opened, please do that and convert it to a Discussion.
